test/sales_engine_test.rb

```diff
@@ -0,0 +1,13 @@
+require './test/test_helper'
+require './lib/sales_engine'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.